rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/config/ArchiveIdentificationConfigLoadingRuleProvider.java

```diff
@@ -40,38 +41,71 @@ public ArchiveIdentificationConfigLoadingRuleProvider()
     }
 
     @Inject
-    private CompositeChecksumIdentifier identifier;
+    private CompositeArchiveIdentificationService identifier;
 
     @Override
     public Configuration getConfiguration(final GraphContext grCtx)
     {
-        ConfigurationBuilder config = ConfigurationBuilder.begin();
-        config.addRule().perform(new GraphOperation()
+        return ConfigurationBuilder.begin()
+                    .addRule()
+                    .perform(new AddSortedFileIndexOperation())
```

Comment:
I guess this is still useful since people can add their own hashes. But can we really expect them to be sorted?